examples/offline_inference_neuron.py

```diff
@@ -0,0 +1,33 @@
+import os
+from vllm import LLM, SamplingParams
+
+# Sample prompts.
+prompts = [
+    "Hello, my name is",
+    "The president of the United States is",
+    "The capital of France is",
+    "The future of AI is",
+]
+# Create a sampling params object.
+sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
+
+os.environ['MASTER_ADDR'] = 'localhost'
+os.environ['MASTER_PORT'] = '12355'
+os.environ['NEURONX_DUMP_TO'] = os.path.join(os.getcwd(), "_compiler_cache")
+os.environ["NEURON_CC_FLAGS"] = " -O1 "
+
+# Create an LLM.
+llm = LLM(model="openlm-research/open_llama_3b",
+          tensor_parallel_size=2,
+          max_num_seqs=8,
+          max_model_len=128,
+          block_size=128,
+          device="cpu")
```

Comment:
Why does the device need to be "cpu" here?

Reply:
What are the constraints on `block_size` and `max_model_len` and `max_num_seqs`? If there's any, please specify it.

Reply:
1) removed the device='cpu' here, and the device can be automatically detected.

Reply:
Also, specified the constraints on supporting neuron device explicitly